docs/data/data-grid/column-menu/ColumnMenuGridReferencesNoSnap.tsx

```diff
@@ -131,7 +131,7 @@ const rows = [
   },
 ];
 
-export default function ColumnMenuGridReferences() {
+export default function ColumnMenuGridReferencesNoSnap() {
```

Comment:
While this demo is using the data grid, it's part of the docs and there seems to be no practical value of testing visual regressions in it.